.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -70,3 +70,18 @@ Documentation change PR (review please): https://github.com/status-im/status.im/
 | Please embed Image/Video here of the before and after.  | Please embed Image/Video here of the before and after.  | Please embed Image/Video here of the before and after. |
 
 status: ready <!-- Can be ready or wip -->
+
+<!-- Uncomment this section for status-go upgrade/dogfooding pull requests
+
+- Specify potentially impacted user flows in _Areas that maybe impacted*.
+- Ensure that _Steps to test_ is filled in.
+
+### Risk
+
+Described potential risks and worst case scenarios.
+
+Tick **one**:
+- [ ] Low risk: 2 devs must perform testing as specified above and attach their results as comments to this PR.
+- [ ] High risk: QA team perform additional testing in the specified affected areas.
```

Comment:
I'd add here a note that merging should not be allowed for High-risk PRs without QA approves it.

Reply:
done!